packages/amplify-codegen/src/callbacks/prePushAddCallback.js

```diff
@@ -16,11 +16,15 @@ async function prePushAddCallback(context, resourceName) {
 
   let shouldGenerateCode = false;
   if (context.exeInfo.inputParams) {
-    normalizeInputParams(context);
-    const inputParams = context.exeInfo.inputParams[constants.Label];
-    const yesFlag = context.exeInfo.inputParams.yes;
+    const frontend = getFrontEndHandler(context);
 
-    shouldGenerateCode = await determineValue(inputParams, yesFlag, 'generateCode', true, () => askShouldGenerateCode());
+    if (frontend !== 'flutter') {
```

Comment:
Is this skip also required when codegen is already setup in this line: 13?

Reply:
To my knowledge the only way to configure codegen is to run the amplify codegen add command, which yields the following:
```
==> amplify-dev codegen add
flutter is not supported by codegen plugin
```

Therefor line 13 should never be executed in a flutter project, correct?